tasks/libs/pipeline_notifications.py

```diff
@@ -80,7 +80,7 @@ def read_owners(owners_file):
     from codeowners import CodeOwners
 
     with open(owners_file, 'r') as f:
-        return CodeOwners(f.read())
+        return CodeOwners(f.read().rstrip())
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why do we need to `rstrip` the file ? Just wondering since it was already working before !

Reply:
I will try removing this and see if it works! was trying to be safe